src/main/java/com/klarna/hiverunner/config/HiveRunnerConfig.java

```diff
@@ -153,8 +155,8 @@ public String getHiveExecutionEngine() {
      * Determines the statement parsing behaviour of the interactive shell. Provided to emulate slight differences
      * between different clients.
      */
-    public CommandShellEmulation getCommandShellEmulation() {
-        return CommandShellEmulation.valueOf(getString(COMMAND_SHELL_EMULATION_PROPERTY_NAME).toUpperCase());
+    public CommandShellEmulator getCommandShellEmulation() {
```

Comment:
Renaming these methods wouldn't be a problem since the return type change has already broken backwards compat.